src/couch_replicator/src/couch_replicator_docs.erl

```diff
@@ -456,26 +456,26 @@ maybe_add_trailing_slash(Url) ->
 make_options(Props) ->
     Options0 = lists:ukeysort(1, convert_options(Props)),
     Options = check_options(Options0),
-    DefWorkers = config:get("replicator", "worker_processes", "4"),
-    DefBatchSize = config:get("replicator", "worker_batch_size", "500"),
-    DefConns = config:get("replicator", "http_connections", "20"),
-    DefTimeout = config:get("replicator", "connection_timeout", "30000"),
-    DefRetries = config:get("replicator", "retries_per_request", "5"),
+    DefWorkers = config:get_integer("replicator", "worker_processes", 4),
+    DefBatchSize = config:get_integer("replicator", "worker_batch_size", 500),
+    DefConns = config:get_integer("replicator", "http_connections", 20),
+    DefTimeout = config:get_integer("replicator", "connection_timeout", 30000),
+    DefRetries = config:get_integer("replicator", "retries_per_request", 5),
     UseCheckpoints = config:get("replicator", "use_checkpoints", "true"),
```

Comment:
Do you want to convert this into `config:get_boolean/3` and get rid of `list_to_existing_atom(UseCheckpoints)`?

Reply:
Changed.